src/Libraries/CoreNodeModels/Logic/AndOr.cs

```diff
@@ -60,7 +60,7 @@ protected override void RemoveInput()
 
         protected override string GetInputName(int index)
         {
-            return "bool" + index;
+            return "boolean" + index;
```

Comment:
Did we end up going with "boolean" or "bool", @Amoursol? I see "boolean" used here but "bool" used above?

Reply:
@aparajit-pratap as per https://autodesk.slack.com/archives/CPJ5UQW0Z/p1599597480011600, we agreed to have **long form** for all port names, except the _commonly used DesignScript base type abbreviations_. In this case please use `bool` @martinstacey with `boolean` added inside the hover description if not already there.

Reply:
Adressed in 29402fa